crates/execution/src/order_emulator/emulator.rs

```diff
@@ -1052,88 +1052,75 @@ impl OrderEmulator {
         }
     }
 
+    #[allow(clippy::too_many_lines)]
```

Comment:
Interesting there is this lint, I haven't seen it before. Since this is WIP it's fine to allow for now.